### ISSUE TITLE
Bring back center snapping and fix shortcut bug

### ISSUE THREAD

**Issue by bmattb** [swissspidy/web-stories-wp#11053]
## Bug Description

Since updating to 1.16 there have been complaints about snapping no longer working as expected, and this was also observed in yesterday's bug-bash.

In my testing I have noted the following:
1. this issue seems to happen early in story creation
2. This issue seems to self resolve sometimes
3. This issue is very challenging to consistently reproduce.
4. This issue has existed since v1.16 at least

Issues found:
1. Center snapping guides do not appear in the editor consistently
2. Object snapping doesn't seem to occur consistently
3. Placing an image as background by using edge fill fails

Discussion with another XWP developer it was noted:

> Cached assets probably.
First time or two it doesn't load, but then all the assets are loaded and cached and then it does.

Here are some examples of my findings: (all in chrome on macos)

**v1.16 snapping test:** Not working 
https://images.zenhubusercontent.com/235435637/6358a2d5-cf3c-43e6-a4a1-a90ee6330526/snapping_test___v1_16.mov

**v1.18.1 snapping test:** Working
https://images.zenhubusercontent.com/235435637/a82d641a-27bf-4f40-9477-9bee7a53dadf/snapping_test___v1_18_1.mov
## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

**Comment by merapi** [comment]
After testing I have my theory about what sometimes happens here, to verify that theory we asked anyone facing this issue (no snapping lines at all) to press the `Disable snapping and drop targeting (Hold ⌘)` shortcut **once** and see if that brings back the snapping.

**Comment by bmattb** [comment]
closing as we have identified a workaround and are waiting on feedback from Newscorp to confirm rca

**Comment by bmattb** [comment]
@merapi here is the user describing the issue

https://user-images.githubusercontent.com/58738530/160860674-0c2ffea7-5850-48b0-823a-df5a9313e193.mov

e

**Comment by bmattb** [comment]
AS noted by @merapi the movable library [was updated in Nov 21](https://github.com/daybrush/moveable/releases/tag/0.27.0) and this removed snapcenter replacing it with snapDirections, which adversely impacted center snap guidelines.

**Comment by felipebochehin87** [comment]
Tested against https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/, using PR #11136.

1. Created a new story
2. Added items and dragged them all around in the canvas
  * When I reach the vertical center of the canvas, center snapping is displayed
  * Dragging the image to the edges and dropping them there adds the images as background image
3. Sticky mode works as expected too

**Comment by bmattb** [comment]
Thanks for this @merapi
